packages/components/src/focal-point-picker/styles/focal-point-style.ts

```diff
@@ -6,18 +6,25 @@ import styled from '@emotion/styled';
 export const PointerCircle = styled.div`
 	background-color: transparent;
 	cursor: grab;
-	height: 48px;
-	margin: -24px 0 0 -24px;
+	height: 40px;
+	margin: -20px 0 0 -20px;
 	position: absolute;
 	user-select: none;
-	width: 48px;
+	width: 40px;
 	will-change: transform;
 	z-index: 10000;
-	background: rgba( 255, 255, 255, 0.6 );
+	background: rgba( 255, 255, 255, 0.4 );
+	border: 1px solid rgba( 255, 255, 255, 0.3 );
 	border-radius: 50%;
-	backdrop-filter: blur( 4px );
-	box-shadow: rgb( 0 0 0 / 20% ) 0px 0px 10px;
+	backdrop-filter: blur( 16px ) saturate( 180% );
+	box-shadow: rgb( 0 0 0 / 8% ) 0px 0px 8px;
+	transition: transform 100ms linear;
```

Comment:
I think we need `reduceMotion` here as well.

Reply:
Added in https://github.com/WordPress/gutenberg/pull/58459/commits/e4ee0d84e5375a9b95f7901de08643d8342c384c.